src/components/MyKiva/BadgeModal.vue

```diff
@@ -1,5 +1,21 @@
 <template>
-	<KvLightbox :visible="show" :title="title" @lightbox-closed="closeLightbox">
+	<KvLightbox
+		:class="{'badge-modal': !isJourneyActive && !isEarnedSection}"
```

Comment:
Does the earned badge state of the modal look correct in regards to the header buttons without these new styles?